packs/smart_items/assets/rising_pillar_temple/data.json

```diff
@@ -1,6 +1,6 @@
 {
-  "id": "c9101f29-69f7-4f3b-8739-3d0a33651969",
+  "id": "0bc5bfff-e355-4258-9c11-c04f2cbe4f99",
```

Comment:
is there a reason why the id changed?

Reply:
I think this ID is autogenerated when I create the item as a Custom Item. Didn't add or change it on purpose
I think we should remove this field, right?